packages/typography/src/FormElements/Label.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { HTMLElementProps } from '@leafygreen-ui/lib';
+import { BaseFontSize, fontFamilies, typeScales } from '@leafygreen-ui/tokens';
+import { Mode } from '../types';
+import { palette } from '@leafygreen-ui/palette';
+import { useBaseFontSize } from '@leafygreen-ui/leafygreen-provider';
+
+const labelStyle = css`
+  font-family: ${fontFamilies.default};
+  font-weight: bold;
+`;
+
+const labelTypeScale: Record<BaseFontSize, string> = {
+  [BaseFontSize.Body1]: css`
+    font-size: ${typeScales.body1.fontSize};
+    line-height: ${typeScales.body1.lineHeight};
+  `,
+  [BaseFontSize.Body2]: css`
+    font-size: ${typeScales.body2.fontSize};
+    line-height: ${typeScales.body2.lineHeight};
+  `,
+};
+
+const labelColorStyle: Record<Mode, string> = {
+  [Mode.Light]: css`
+    color: ${palette.black};
+  `,
+  [Mode.Dark]: css`
+    color: ${palette.white};
+  `,
+};
+
+const disabledLabelColorStyle: Record<Mode, string> = {
+  [Mode.Light]: css`
+    color: ${palette.gray.base};
+  `,
+  [Mode.Dark]: css`
+    color: ${palette.gray.light1};
+  `,
+};
+
+type LabelProps = HTMLElementProps<'label', never> & {
+  darkMode?: boolean;
+  htmlFor: string;
+  disabled?: boolean;
+};
```

Comment:
🤷 Could we move these types for both Description and Label to `types.ts`?

Reply:
In principle, yes, but I feel like these interfaces are small enough that they can live here. If I were to move them I'd prefer to refactor each component into its own directory, for example `FormElements/Label/types.ts` etc

Reply:
I like that idea of having separate directories! I'll leave the call up to you.

Reply:
Again, I think in principle yes, but for a small component like this I think it's easier to keep it all in one file. The threshold is blurry for when a component should be broken into multiple files